packages/ember-utils/lib/empty-object.js

```diff
@@ -1,8 +1,9 @@
-// This exists because `Object.create(null)` is absurdly slow compared
-// to `new EmptyObject()`. In either case, you want a null prototype
-// when you're treating the object instances as arbitrary dictionaries
-// and don't want your keys colliding with build-in methods on the
-// default object prototype.
+// This exists because `Object.create(null)` WAS slow compared to `new
```

Comment:
We should likely remove this file completely. I did an ember-observer code search ([here](https://emberobserver.com/code-search?codeQuery=empty-object)) and there are no direct usages of our version any longer (though there are addons that have made their own copies of this code).